src/app/learn/get-started/advanced/localization/index.html

```diff
@@ -4,18 +4,154 @@
 
   <stache-page-summary>
     <p>
-      SKY UX Builder provides the capability to declare localization strings <a routerLink="/learn/reference/naming-and-placement">in a JSON file </a> and then reference those strings throughout your SPA using the <stache-code>skyAppResources</stache-code> pipe or the <stache-code>SkyAppResourcesService</stache-code> service in <stache-code>@skyux/i18n</stache-code>.
+      SKY UX Builder provides two methods for localization, <stache-code>skyLibResources</stache-code> for libraries and <stache-code>skyAppResources</stache-code> for SPAs. 
     </p>
   </stache-page-summary>
 
   <sky-alert alertType="info">
     SKY UX Builder version <stache-code>1.14</stache-code> or later is required for a library to provide its own resource files. The consuming SPA must also be on version <stache-code>1.14</stache-code> or later.
   </sky-alert>
 
+<stache-page-anchor>
+  Localization for libraries
+</stache-page-anchor>
+
+<p>
+  Localization strings for libraries must be set when publishing to NPM because they are not fetched at runtime as they are for SPAs. 
+</p>
+
+<h3>
+  Install the plugin
+</h3>
+
+<ol>
+  <li>
+    <p>
+      Install the SKY UX builder plugin.
+    </p>
+      <sky-code-block>
+        npm install --save-dev --save-exact @skyux-sdk/builder-plugin-skyux
+      </sky-code-block>
+    </li>
+  <li>
+    <p>
+      Add the plugin to your <stache-code>skyuxconfig.json file</stache-code>.
+    </p>
+      <sky-code-block>
+        {
+          "plugins": [
+            "@skyux-sdk/builder-plugin-skyux"
+          ]
+        }  
+      </sky-code-block>
+  </li>
+</ol>
+
+<h3>
+  Create a resources provider
+</h3>
+
+  <ol>
+    <li>
+      <p>
+        Create a new directory named <stache-code>plugin-resources</stache-code> in <stache-code>./src/app/public/plugin-resources</stache-code>. The name and location of the directory are important.
+      </p>
+    </li>
+    <li>
+      <p>
+        Inside the <stache-code>plugin-resources</stache-code> directory, create a new file named <stache-code>my-library-resources-provider.ts</stache-code> and replace <stache-code>my-library</stache-code> with the name of your library. In the sample below, replace <stache-code>MyLibrary</stache-code> with the name of your library.
```

Comment:
I'd change "Inside" to "In" because it means the same thing and is shorter.

Also, I'd delete "new" again because it's redundant and I'd rearrange the bit after the opening clause to "create a `my-library-resources-provider.ts` file..." Also in that sentence, I'd change "and replace" to "where" and "with" to "is."

It might juts be me, but it seems like the third sentence should be fleshed out and maybe even made into a separate step. Currently, we tell them to replace something from the code block, but we've never told them to enter the code block in the file they created. It seems like we should have something along the lines of "In the new file, import `SkyAppLocaleInfo` and `SkyLibResourcesProvider` from the `@skyux/i18n` library and export a `MyLibraryResourcesProvider` class where `MyLibrary` is the name of your library." Or something like that.